src/Components/ArticleCard.jsx

```diff
@@ -1,20 +1,22 @@
-import {useState} from "react"
-
-function ArticleCard({article}) {
-    return(
-        <section>
-            <h3>{article.title}</h3>
-            <img src={article.article_img_url} alt={article.title} />
-            <ul>
-                <li>Author: {article.author}</li>
-                <li>Topic: {article.topic} </li>
-                <li>Date: {article.created_at}</li>
-                <li>Likes: {article.votes}</li>
-                <li>Comments: {article.comment_count}</li>
-            </ul>
-        </section>
-    )
+import { Link } from "react-router-dom";
 
+function ArticleCard({ article }) {
+    // console.log(article)
+  return (
+    <section className="article-card">
+      <Link to={`/articles/${article.article_id}`}>
+        <h3>{article.title}</h3>
+        <img src={article.article_img_url} alt={article.title} />
+      </Link>
+      <ul>
+        <li>Author: {article.author}</li>
+        <li>Topic: {article.topic} </li>
+        <li>Date: {article.created_at}</li>
+        <li>Likes: {article.votes}</li>
+        <li>Comments: {article.comment_count}</li>
+      </ul>
```

Comment:
I'd have a quick read of this https://www.refactoringui.com/previews/labels-are-a-last-resort